### ISSUE TITLE
Game crashing exit code 1

### ISSUE THREAD

**Issue by kmayb** [MrCrayfish/Controllable#418]
I'm trying to use controllable 1.20.1 0.20.3 

I'm running Forge 1.20.1-47.03 with Framework 0.6.5, as well as Optifine 1.20.1_HD_U-_5_pre6 (Don't know if optifine would cause any issues). Game works fine running those three, but crashes once controllable is in the mods folder.

**Comment by MrCrayfish** [comment]
Controllable 0.20.3 requires Forge 47.0.4 or above. You'll have to wait for Optifine to update.

**Comment by Jam4Mad** [comment]
Thanks for this post I have been trying for a few days now to get it to work based on videos I have found. I also tried going back to a few versions of controllable that support 1.20.1 but still got the same error. And to rule out user error on my side is this an expected result? e.g., using controllable 20.0, 1, 2 and getting the same error.

**Comment by Nuanulla** [comment]
[latest.log](https://github.com/MrCrayfish/Controllable/files/11924462/latest.log)

Attaching my log here. Game runs fine with either OptiFine OR Controllable. Not both at the same time. Framework does not appear to cause a crash. Exit code 1 on Windows. Exit code 2 on Deck via Prism, beware of bug report dupes.
Forge 47.0.35
JDK 7.0.3 built-in
OptiFine I5 pre7
Controllable 0.20.3
Framework 0.6.6

**Comment by Jam4Mad** [comment]
Thanks for the info Nuanulla, I still see the issue on my Mac M1. 
Java JRE 1.8
forge-1.20.1-47.0.34-installer
controllable-forge-1.20.1-0.20.3

<img width="854" alt="2023-07-01_15-01-10" src="https://github.com/MrCrayfish/Controllable/assets/87128687/7e97d0b8-d4c6-45e8-a3ba-20be411f3d70">

**Comment by Nuanulla** [comment]
@Jam4Mad Isn't your issue different? You're missing the Framework requirement as it's stated in your screenshot.
https://www.curseforge.com/minecraft/mc-mods/framework

**Comment by mysteryman23** [comment]
Attaching my log here. Game runs fine with either OptiFine OR Controllable. Not both at the same time. Framework does not appear to cause a crash. Exit code 1 on Windows. Exit code 2 on Deck via Prism, beware of bug report dupes.
Forge 47.0.35
JDK 7.0.3 built-in
OptiFine I5 pre7
Controllable 0.20.3
Framework 0.6.6

I agree and see what you're saying Nuanulla. Is there anyway around this currently with them not working together? I feel I've tried every version and every possible thing to get it to work. They just aren't matching up together to work at the same time.

**Comment by Jam4Mad** [comment]
@Nuanulla Thank you, and you are correct it was the framework. I just installed it according to your link and controllable is now working. I am new to Java/mods so I was unsure what the framework was... part of the controllable or forged and the youtube videos I found did not speak to the framework. And now I know better, thanks again :)

**Comment by k5TRY** [comment]
Same here when using optifine or controllable works fine ... when using both game crashes exit code 1.

![latest-crash.txt](https://github.com/MrCrayfish/Controllable/files/11987852/latest-crash.txt)

**Comment by Llakatasan** [comment]
Same here, Controllable seems to be incompatible with Optifine as of now in 1.20.1

**Comment by MiguelTheGamer0** [comment]
@Llakatasan Not really. I took out Optifine out of my mods folder and I still crash upon launching the game. I have the latest version of Controllable (0.20.3) and Framework (0.6.6).

**Comment by KSUMatt** [comment]
Any update on this? I'd love to use both.

**Comment by MrCrayfish** [comment]
Fixed in https://github.com/MrCrayfish/Controllable/commit/39bb8d72f5f5cb3c94466aee663d43e681b27ae7
